pylot/loggers/chauffeur_logger_operator.py

```diff
@@ -182,3 +201,58 @@ def on_can_bus_update(self, msg):
                                                    msg.timestamp.coordinates[0])
         with open(file_name, 'w') as outfile:
             json.dump(str(msg.data.forward_speed), outfile)
+
+    def on_traffic_lights_update(self, msg):
```

Comment:
IMHO, this code should not be in a callback because you're not using the message at all. Instead, the code should be invoked from the on_top_down_camera_update.